src/lib/migration.ts

```diff
@@ -0,0 +1,64 @@
+export const defaultLegacyVolume = () => 'resin-data';
+
+export function singleToMulticontainerApp(app: any) {
```

Comment:
Can you make this more specific please (at a minimum `Dictionary<any>`) - if it truly can be `any` then there's a whole bunch of places where the code will blow up

Reply:
I've correctly typed the output, the input isn't as necessary so as you say I've added `Dictionary<any>`